src/components/map/ha-location-editor.ts

```diff
@@ -127,9 +127,11 @@ class LocationEditor extends LitElement {
   }
 
   private async _initMap(): Promise<void> {
+    const mql = matchMedia("(prefers-color-scheme: dark)");
+
     [this._leafletMap, this.Leaflet, this._tileLayer] = await setupLeafletMap(
       this._mapEl,
-      this.hass.themes?.darkMode,
+      this.hass.themes?.darkMode || mql.matches,
```

Comment:
can we pass this from `onboarding-core-config` instead of adding a media query here just for that case?

Reply:
Also this would now override if the theme is set to dark manually?

Reply:
Probably... I'll have to do some more tests for that!